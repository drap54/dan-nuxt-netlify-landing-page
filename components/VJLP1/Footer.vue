<template>
    <footer>
        This is the footer section!
    </footer>
</template>