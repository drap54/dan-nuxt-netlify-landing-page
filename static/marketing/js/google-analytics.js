var optimizeId = document.getElementById('ga_data').getAttribute('go_id') || 'OPT-PHSNXP6';
var analyticsId = document.getElementById('ga_data').getAttribute('ga_id') || 'UA-142143961-1';

(function (i, s, o, g, r, a, m) {
  i['GoogleAnalyticsObject'] = r; i[r] = i[r] || function () {
    (i[r].q = i[r].q || []).push(arguments)
  }, i[r].l = 1 * new Date(); a = s.createElement(o),
    m = s.getElementsByTagName(o)[0]; a.async = 1; a.src = g; m.parentNode.insertBefore(a, m)
})(window, document, 'script', 'https://www.google-analytics.com/analytics.js', 'ga');

ga('create', analyticsId, 'auto');
ga('require', optimizeId);
// ga('send', 'pageview');