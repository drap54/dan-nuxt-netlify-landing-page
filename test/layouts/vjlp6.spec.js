
import { mount } from '@vue/test-utils'
import _ from 'lodash'
import VJLP6 from '@/layouts/vjlp5.vue'
import Footer from '@/components/Base/TheFooter.vue'
import { retrieveFiles, retriveFrontMattertoJSON } from '@/test/utils/fileUtil.js'

const VJLP6_DIR = '/assets/content/landing-page/marketing/vjlp5/'

describe('Testing VJLP5 layout vjlp5.vue', () => {
  let wrapper
  beforeAll(async () => {
    const md = _
      .chain(await retrieveFiles(VJLP6_DIR, '.md'))
      .map(files => VJLP6_DIR + files)
      .sample()
      .value()

    const { html, attributes } = retriveFrontMattertoJSON(md)
    wrapper = mount(VJLP6, {
      stubs: ['nuxt']
    })

    await wrapper.setData({
      mdData: attributes,
      htmlBody: html
    })
  })

  test('Renders correctly', () => {
    expect(wrapper.element).toMatchSnapshot()
  })

  test('VJLP6 is a Vue instance', () => {
    expect(wrapper.vm).toBeTruthy()
  })

  test('VJLP6 has Footer Component', () => {
    const footer = wrapper.findComponent(Footer)
    expect(footer.exists()).toBeTruthy()
  })

  test.skip('Join and deposit button redirect URL should be https://verajohn.com/#join', () => {
    wrapper.findAll('button').wrappers.forEach((btn) => {
      const _btn = btn.find('a').html()
      expect(_btn.includes('href="https://verajohn.com/#join"')).toBeTruthy()
    })
  })
})
