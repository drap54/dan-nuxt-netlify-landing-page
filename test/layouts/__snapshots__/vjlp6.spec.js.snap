// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing VJLP5 layout vjlp5.vue Renders correctly 1`] = `
<div
  class="vjlp5_wrapper"
>
  <nuxt-stub />
   
  <div
    class="vjlp5-main"
  >
    <header
      id="header"
    >
      <div
        class="container"
      >
        <div
          class="logo"
        >
          <img
            alt="alt logo"
            data-not-lazy=""
            src="@/assets/images/vjlp5/vj-logo.png"
          />
        </div>
         
        <div
          class="nav-links"
        >
          <button
            class="warning"
          >
            <a
              href="https://verajohn.com/#join"
            >
              
              登録
            
            </a>
          </button>
           
          <button
            class="error animated pulse infinite"
          >
            <a
              href="https://verajohn.com/#join"
            >
              <strong>
                入金ボーナス獲得する
              </strong>
            </a>
          </button>
        </div>
      </div>
    </header>
     
    <section
      id="hero"
      style="--bg-image: url('/marketing/img/306-bvj-lan-jp-pro-livecasinov2-0520_banner-2.jpeg'); --bg-image-m: url('/marketing/img/306-bvj-lan-jp-pro-livecasinov2-0520_banner-2.jpeg'); --bg-banner: url('/marketing/img/306-bvj-lan-jp-pro-livecasinov2-0520_banner-2.jpeg');"
    >
      <div
        class="container"
      >
        <div
          class="banner"
        />
      </div>
    </section>
     
    <section
      id="steps"
    >
      <div
        class="container column-3"
      >
        <div
          class="animated fadeIn delay-halfs"
        >
          <span
            class="num"
          >
            1
          </span>
           
          <div>
            <h3>
              ご登録
            </h3>
             
            <p>
              安全 & 安心！
            </p>
          </div>
        </div>
         
        <div
          class="animated fadeIn delay-1s"
        >
          <span
            class="num"
          >
            2
          </span>
           
          <div>
            <h3>
              ご入金
            </h3>
             
            <p>
              ボーナス獲得！
            </p>
          </div>
        </div>
         
        <div
          class="animated fadeIn delay-1-and-halfs"
        >
          <span
            class="num"
          >
            3
          </span>
           
          <div>
            <h3>
              プレイ!
            </h3>
             
            <p>
              お楽しみください！ 
            </p>
          </div>
        </div>
      </div>
    </section>
     
    <section
      id="section1"
    >
      <div
        class="container"
      >
        <div>
          <div>
            <h2>
              ライブカジノで一攫千金ッ！
            </h2>
             
            <p>
              いつでもどこでも、本場カジノの臨場感が楽しめるライブカジノ。ディーラーの動きがリアルタイムで見えるので、イカサマの心配もありません。しかも、還元率は他のギャンブルよりもはるかに高いともいわれています。パチスロの一般的な還元率は約80%と言われていますが、オンラインカジノでは、なんと最低でも「95％」はあるんです！さぁ、日本No.1オンラインカジノのベラジョンで、今をときめく最先端のライブカジノを堪能しましょう！
            </p>
          </div>
           
          <div
            class="separator"
          />
        </div>
      </div>
    </section>
     
    <section
      id="section2"
    >
      <div
        class="container column-2"
      >
        <div>
          <div>
            <h2>
              勝負金が2倍！最高$500初回入金ボーナスをもらってお得にプレイ！
            </h2>
             
            <p>
              ベラジョンでの初回入金時には、軍資金が2倍になる★ビギナーズボーナス★がお待ちしています！最高$500までもらえる、100%入金ボーナスを受け取って、いろんなゲームにトライしてみましょう！ さらに、キャンペーンメール限定ボーナスもご用意していますので、アカウント登録後、キャンペーンメールの配信設定をオンにすることをお忘れなく♪
            </p>
          </div>
          <div>
            <h2>
              安心・安全のオンラインカジノ環境
            </h2>
             
            <p>
              私たちは、ベラジョンカジノが革新的で安心・安全なオンラインカジノであり、食パン以来の大発明★と自負しています！経験豊富なプロ集団が、世界一楽しいエンターテイメントを目標に掲げて、最高のカジノ体験をお届けします！ さらに、各国の政府公認ライセンスを取得し、業界でトップクラスのゲーム配信会社を導入。カジノで遊べる製品は、ランダム・ナンバー・ジェネレーターと呼ばれる、ゲーム結果をランダムに生成するシステムを利用しており、ゲームの公平性も第三者機関によって保証されています。 また、プレイヤーの皆様には、ご自身のゲームプレイ習慣を理解し、健全にカジノをお楽しみいただきたいと思っています。当サイトでご利用いただける「自己規制」機能で、サイトへのアクセス制限や入金に上限を設定することが可能です。安心・安全に、楽しくカジノライフを始めちゃおう！ 
            </p>
          </div>
          <div>
            <h2>
              ご褒美プログラムとベラジョン・ショップ
            </h2>
             
            <p>
              ベラジョンでは、ログインや入金、アカウント認証など、サイト上で行うあらゆるアクションにより、ご褒美としてコインがもらえます。これらのコインは、ベラジョン・ショップでのお得なアイテムを購入時に使用できちゃいます！コインが増えるとレベルも更新され、入金ボーナス、フリースピン、特定のゲームで利用できるボーナスなど、様々なアイテムの購入が可能になります。期間限定アイテムやお得なアイテムも盛りだくさん！中には、キャッシュに変換できるコインもありますよ！ベラジョン・ショップでは様々な人気アイテムを多数取り揃えていますので、ぜひ合わせてご利用ください♪
            </p>
          </div>
        </div>
         
        <div>
          <div>
            <h2>
              完全生配信！ライブカジノゲーム
            </h2>
             
            <p>
              テーブルゲームファンに特におすすめ！ブラックジャックやルーレット、バカラやビデオポーカーなど、バライティ豊かなゲームをと揃えています♪しかも、ライブカジノだから、リアルタイムでディーラーと対戦できるんです！ 実際にチャットを通じて会話までできちゃうので、まるで海外カジノのテーブルに座り、実際にディーラーと対戦してるかのような感覚を味わえます！ なかには、本場の海外カジノから直接配信されているテーブルもあるんです！日本に居ながらにして本場カジノがお手軽に体験できちゃう、オンラインカジノならではのライブゲームはクセになること間違いなし★
            </p>
          </div>
          <div>
            <h2>
              素晴らしいスロットの数々
            </h2>
             
            <p>
              リムジンに乗り込み、シャンパンの栓を抜くことを想像しながら、人気スロットゲームをプレイし始めませんか？パチスロ風スロットの元祖、Hawaiian Dream、クレイジーなゲーム体験を求めて宇宙に旅立つStarburst、さらに一攫千金ジャックポットゲームなど。今すぐプレイして、お気に入りゲームを見つけてみてください！もちろん、パソコン、モバイルなど、利用端末に関わらず、いつでもどこでも最高のゲームをお楽しみいただけます！
            </p>
          </div>
          <div>
            <h2>
              ライブチャット＆Eメール日本語サポート
            </h2>
             
            <p>
              ベラジョンの日本語カスタマーサポートは、迅速かつ丁寧！何かお困りのことがありましたら、お気軽にお問い合わせください♪ 多くのプレイヤーからお寄せいただく質問は、ヘルプ・センタ―のよくある質問(FAQ)または「ヘルプ」よりご確認いただけます。
            </p>
          </div>
          <div>
            <h2>
              ものの5分でスピーディー! 今すぐ登録しよう!
            </h2>
             
            <p>
              最高の環境が整ったベラジョンオンラインカジノに登録するなら今でしょ！ しかも、ベラジョンでの初めてのご入金時には、勝負金が2倍になる★初回入金ボーナス★が、最大$500までもらえます！ 安心・安全・楽しいベラジョンカジノで、今日こそカジノライフをスタートしちゃいましょう！グッドラック！
            </p>
          </div>
        </div>
      </div>
    </section>
     
    <section
      id="section3"
    >
      <div
        class="container"
      >
        <div
          class="separator"
        />
      </div>
    </section>
     
    <section
      id="section4"
    >
      <div
        class="container"
      >
        <div>
          <h2>
            ビギナーズボーナス利用規約
          </h2>
           
          <ol>
            <li>
              
              このキャンペーンでは、100%の初回入金ボーナスを最高$500まで、50%のセカンド入金ボーナスを最高$250まで、100%のサード入金ボーナスを最高$200まで獲得することができます。
            
            </li>
            <li>
              
              有効な入金額により、ボーナスが自動的に進呈され、ボーナスのお引き出しには、ボーナス額の20倍にあたる金額をゲームで賭ける必要があります。
            
            </li>
            <li>
              
              ビギナーズボーナスを獲得するためには、入金時に、入金ページにある「ボーナス」のボックスにチェックが入力されている必要があります。 
            
            </li>
            <li>
              
              ボーナスの有効期限は30日間です。 
            
            </li>
            <li>
              
              このキャンペーンは各入金毎にお1人様1回のみ有効となります。 
            
            </li>
            <li>
              
              このキャンペーンは他のキャンペーンとの併用はできません。
            
            </li>
            <li>
              
              このキャンペーンは日本のプレイヤー様のみ対象となります。 
            
            </li>
            <li>
              
              ボーナスを使ったゲームプレイでの1度の最大賭け額は、スロット/スクラッチカードで$6.25、またはテーブルゲーム/ライブカジノまで$25とします。この金額を上回った場合には、ボーナスとそのボーナスに関連するすべての勝利金は没収されます。
            
            </li>
            <li>
              
              ベラジョンはいかなるときも通告なしでこのキャンペーンを変更、終了できる権利を有します。
            
            </li>
          </ol>
           
          <div
            class="separator"
          />
        </div>
      </div>
    </section>
     
    <div>
      <section
        id="section5"
      >
        
    
        <div
          class="container"
        >
          
        
          <div>
            
            
            <h2>
              最高$500初回入金ボーナス利用規約
            </h2>
            
            
            <ol>
              
                
              <li>
                このキャンペーンは、ベラジョン初めての入金を行うプレイヤー様が対象となります。
              </li>
              
                
              <li>
                このキャンペーンでは、最初の1回の入金で、100％入金ボーナスを獲得することができます。
              </li>
              
                
              <li>
                初回入金ボーナスは、最大$500まで獲得することができます。
              </li>
              
                
              <li>
                入金ボーナスを獲得するには、入金時にボーナスの受け取り希望欄にチェックを入れる必要があります。
              </li>
              
                
              <li>
                入金ボーナスは、$10以上を入金した後、自動的に進呈されます。
              </li>
              
                
              <li>
                入金した分のキャッシュは、ボーナスの出金賭け条件の対象外となります。また、ボーナスは、入金した分のキャッシュの利用後、使用可能となります。
              </li>
              
                
              <li>
                進呈されたボーナスのお引き出しには、ボーナス額の20倍にあたる出金賭け条件をゲームで賭けて満たす必要があります。
              </li>
              
                
              <li>
                出金賭け条件を満たす前に、キャッシュまたはそれに関連するキャッシュ勝利金を出金する場合、残っているボーナスはすべて没収されます。
              </li>
              
                
              <li>
                出金賭け条件は、ボーナス付与から30日以内に達成できない場合、ボーナス及びそれにより獲得した勝利金が没収されます。
              </li>
              
                
              <li>
                ベラジョンはプレイヤー様による不正行為が認められた場合、ボーナスおよびそれにより獲得した勝利金を無効とする権利を有します。不正行為についてはキャンペーンの一般規約をご確認ください。
              </li>
              
                
              <li>
                出金賭け条件への適用率は、ゲームによって異なります。詳細は
                <a
                  href="https://www.verajohn.com/ja/about/our-casino-bonuses"
                >
                  カジノボーナス
                </a>
                のページをご確認ください。
              </li>
              
                
              <li>
                第三者のウェブサイト経由でベラジョンに登録したアカウントは、このボーナスの対象外となります。
              </li>
              
                
              <li>
                このキャンペーンは日本のプレイヤー様のみ対象となります。
              </li>
              
                
              <li>
                ボーナスを使ったゲームプレイでの1度の最大賭け額は、スロット/スクラッチカードで$6.25、またはテーブルゲーム/ライブカジノまで$25とします。この金額を上回った場合には、ボーナスとそのボーナスに関連するすべての勝利金は没収されます。
              </li>
              
                
              <li>
                ベラジョンはいかなるときも通告なしでこのキャンペーンを変更、終了できる権利を有します。
              </li>
              
            
            </ol>
            
            
            <p>
              通常の
              <a
                href="https://verajohn.com/about/terms-and-conditions"
              >
                利用規約
              </a>
              と
              <a
                href="https://verajohn.com/about/promotions-terms-and-conditions"
              >
                キャンペーン一般利用規約
              </a>
              が適用されます。
            </p>
            
            
            <div
              class="separator"
            >
               
            </div>
            
        
          </div>
          
    
        </div>
        

      </section>
      

    </div>
     
    <div>
      <footer>
        <div
          class="container"
        >
          <div>
            <div
              id="footer-icon"
            >
              <a
                aria-label="Responsible Gaming"
                href="https://www.verajohn.com/about/responsible-gaming"
                target="_blank"
              >
                <img
                  alt="alt img"
                  src="https://files.vjpromo.com/bannerflow/lp-core/ico/icon_age.svg"
                />
              </a>
               
              <a
                aria-label="Gambling Theraphy"
                href="https://www.gamblingtherapy.org/en"
                target="_blank"
              >
                <img
                  alt="alt img"
                  src="https://files.vjpromo.com/bannerflow/lp-core/ico/icon_gt.svg"
                />
              </a>
               
              <a
                aria-label="Curacao"
                href="#"
                target="_blank"
              >
                <img
                  alt="alt img"
                  src="https://files.vjpromo.com/bannerflow/lp-core/ico/icon_curacao.svg"
                />
              </a>
            </div>
             
            <p>
              <a
                aria-label="Responsible Gaming"
                href="https://www.verajohn.com/about/responsible-gaming"
                target="blank"
              >
                
            ギャンブルには中毒性があります。 自己責任を持ってプレイしてください。
          
              </a>
            </p>
             
            <p>
              当ウェブサイトは、キュラソー政府の商業登記に登録されている、番号149132のBreckenridge Curacao B.V.（登録住所：Emancipatie Boulevard, Dominico F. “Don” Martina 31, Willemstad, Curaçao）により運営されています。
            </p>
             
            <p>
              Breckenridge Curaçao B.V.はキュラソー州知事により発行されたMaster Gaming License #5536/JAZに基づく C.I.L Curacao Interavtive Licensing N.V. により付与されたサブライセンスを通して正式な許可を所持しています
            </p>
          </div>
        </div>
      </footer>
    </div>
  </div>
</div>
`;
